matico_charts/src/Utils/index.ts

```diff
@@ -18,8 +18,8 @@ export const getBoolOrProperty = (
 };
 
 export const sanitizeColor = (color: ColorOutput) => {
-  if (!color) return null;
-  if (isString(color)) return color;
+  if (!color) return undefined;
+  if (isString(color)) return color as string;
```

Comment:
Do you know the reason that color isn't returning as a `string` without the `as string` suffix declaration?